roles/thelounge/tasks/main.yaml

```
@@ -0,0 +1,48 @@
+- name: Install TheLounge package
+  ansible.builtin.package:
+    name: thelounge
+    state: present
+
+- name: Install acl package
+  ansible.builtin.package:
+    name: acl
+    state: present
```

Only a suggestion, not blocking at all:

```suggestion
- name: Install required packages
  ansible.builtin.package:
    name: 
      - thelounge
      - acl
    state: present
```

---

@sim6: Since this is a pretty trivial improvement, I am applying the suggestion now